constants/styles.js

```diff
@@ -37,3 +37,25 @@ export const presetColors = [
 
 export const baseGridWidth = 300
 export const baseInnerWidth = 768
+
+export const ADMIN_HEADER_STYLES = {
```

Comment:
let's use css where we can. Do we need these style objects?

Reply:
We need them
## Without styles object
<img width="1217" alt="Screen Shot 2023-08-09 at 10 24 46 AM" src="https://github.com/AMP-SCZ/dpdash/assets/19805355/feacd034-d567-4003-b8b2-761f1567a3be">

## with styles
<img width="1217" alt="Screen Shot 2023-08-09 at 10 25 06 AM" src="https://github.com/AMP-SCZ/dpdash/assets/19805355/cb7812a4-1a68-4728-9e53-6f6c4de641ee">

Reply:
we need the styles or we need them to be objects? We should be able to use css